src/WebUI/Utils.ts

```diff
@@ -113,7 +111,7 @@ export class Utils {
         let imageCountMap: { [key: string]: number } = {};
         let imageString: string = "";
         containers.forEach(container => {
-            if (imageCountMap[container.image] == undefined) {
+            if (!imageCountMap[container.image]) {

```

Comment:
why so much complexity, can not we simply say if container length is > 1 then do [0].image and .length, if it is > 0 then return [0] otherwise empty string.

Reply:
Multiple containers can have the same image. So here I am computing the unique images between containers